### PR TITLE
docs:Adding documentation to images.go on api

### PR DESCRIPTION
**Pull request by adeliaferreira**
# Description

Adding Parameters on images.go documentation api

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation update
- [ ] Tests update
- [ ] Refactor

### CODE REVIEW

File: pkg/routes/images.go [hunk 1]
```diff
@@ -286,6 +286,7 @@ func CreateImage(w http.ResponseWriter, r *http.Request) {
 // @Accept       json
 // @Produce      json
 // @Param        body	body	models.Image	true	"request body"
+// @Param		 imageId path int true "Image ID" example(1234)
```
Comment (mgold1234) [inline pkg/routes/images.go:289]: should be align with // other rows

File: pkg/routes/images.go [hunk 2]
```diff
@@ -467,8 +468,13 @@ func ValidateGetAllImagesSearchParams(next http.Handler) http.Handler {
 // @Tags         Images
 // @Accept       json
 // @Produce      json
-// @Param		 required_parm query string true "A placeholder for required parameter" example(cat)
-// @Param		 optional_parm query int false "A placeholder for optional parameter" example(42)
+// @Param		 limit query int false "Return number of images until limit is reached." example(1200)
```
Comment (mgold1234) [inline pkg/routes/images.go:471]: should be align with // other rows

File: pkg/routes/images.go [hunk 3]
```diff
@@ -286,6 +286,7 @@ func CreateImage(w http.ResponseWriter, r *http.Request) {
 // @Accept       json
 // @Produce      json
 // @Param        body	body	models.Image	true	"request body"
```
Comment (mgold1234) [inline pkg/routes/images.go:288]: I think you can delete this row

File: pkg/routes/images.go [hunk 4]
```diff
@@ -757,6 +758,7 @@ func GetMetadataForImage(w http.ResponseWriter, r *http.Request) {
...
+// @Param		 imageId path int true "Image ID" example(1234)
```
Comment (mgold1234) [inline pkg/routes/images.go:761]: align with other rows

**Review by acosferreira — DISMISSED** [review]
lgtm

File: pkg/routes/images.go (hunk 3 above)
Comment (mgold1234) [reply pkg/routes/images.go:288]: @adeliaferreira wdyt?

**Review by ldjebran — APPROVED** [review]
lgtm

**Review by acosferreira — APPROVED** [review]
lgtm

File: pkg/routes/images.go (hunk 3 above)
Comment (ldjebran) [reply pkg/routes/images.go]: @mgold1234 as discussed offline, I think we can resolve this question, what do you think  ?

Comment (mgold1234) [reply pkg/routes/images.go]: Yes, we can

**Review by mgold1234 — APPROVED** [review]
/lgtm

**Pull request merged**